charts/govuk-rails-app/templates/deployment.yaml

```diff
@@ -22,6 +22,8 @@ spec:
           ports:
             - name: http
               containerPort: {{ .Values.appPort }}
+            - name: metrics
+              containerPort: {{ .Values.metricsPort }}
```

Comment:
I wonder if we could define .Values.extraPorts` or something like that? I guess we want all apps to export metrics, so this is probably fine.

Reply:
yeah could do, i'm not really sure of what other ports that there might be coming at the minute, could change it to this then if there are more to add 
extraPorts:
  metricsPort: 9394

Reply:
suppose it will be easy to do later as well